src/serverenvironment.cpp

```diff
@@ -1340,11 +1340,6 @@ void ServerEnvironment::step(float dtime)
 
 	if (m_active_block_modifier_interval.step(dtime, m_cache_abm_interval))
 		do { // breakable
-			if (m_active_block_interval_overload_skip > 0) {
-				ScopeProfiler sp(g_profiler, "SEnv: ABM overload skips");
-				m_active_block_interval_overload_skip--;
-				break;
```

Comment:
Now it no longer needs to be breakable at L1342

Reply:
Good point!